test/performance/search/search_benchmark.cpp

```diff
@@ -335,7 +335,8 @@ seqan::StringSet<range_t> convert_range_of_range(std::vector<std::vector<origina
 
 void seqan2_unidirectional_search(benchmark::State & state, options && o)
 {
-    using reference_index_t = seqan::Index<seqan::DnaString, seqan::FMIndex<void, seqan::FastFMIndexConfig<void, uint32_t>>>;
+//     using reference_index_t = seqan::Index<seqan::DnaString, seqan::FMIndex<void, seqan::FastFMIndexConfig<void, uint32_t>>>;
+    using reference_index_t = seqan::Index<seqan::DnaString, seqan::FMIndex<void>>;
```

Comment:
Do not compare SeqAn3 WT implementation with SeqAn2 EPR implementation. Set both to WT for now.